periph_sw/nRF52_RGB_Strip/nRF52_RGB_Strip.ino

```diff
@@ -4,14 +4,10 @@
 #include "ble_handler.h"
 #include "eeprom_handler.h"
 
-/* Need to undefine min and max in order to compile <String>. */
-#undef max
```

Comment:
Why is this not needed any more?

Reply:
There was a namesapce conflict between libraries. I replaced the calls by constrain() and std::max() where needed.